metadata.json

```diff
@@ -10,6 +10,14 @@
   "issues_url": "https://github.com/voxpupuli/puppet-zypprepo/issues",
   "dependencies": [],
   "requirements": [
+    {
+      "name": "puppetlabs/stdlib",
+      "version_requirement": ">= 4.18.0 < 7.0.0"
+    },
+    {
+      "name": "puppetlabs/concat",
+      "version_requirement": ">= 1.2.5 < 7.0.0"
+    },
```

Comment:
How was decided the lower bound ?

Reply:
The lower bound was copied from the Yum module. As the code is for the most part copied directly I also copied the version requirements from that module. In your opinion should these be different?